Client/Frontend/Settings/ThemeSettings/ThemeSettingsController.swift

```diff
@@ -56,17 +68,85 @@ class ThemeSettingsController: ThemedTableViewController {
                                                object: nil)
     }
 
+    func newState(state: ThemeSettingsState) {
+        themeState = state
+        tableView.reloadData()
+    }
+
+    override func applyTheme() {
+        super.applyTheme()
+    }
```

Comment:
needed?

Reply:
I removed and test it with the flag on and off and everything was working as expected